src/CMakeLists.txt

```diff
@@ -42,6 +47,23 @@ add_executable(Test prog/Test.cpp)
 target_link_libraries(Test Minimizer Models Utility)
 target_compile_features(Test PUBLIC cxx_std_14)
 
+add_executable(CalcGW prog/CalcGW.cpp)
+target_link_libraries(CalcGW Utility GW MinimumTracer TransitionTracer)
+target_compile_features(CalcGW PUBLIC cxx_std_14)
```

Comment:
We could make all cxx_std_14 to cxx_std_17 with this.
You will need it anyway for the filesystem and we require it in the conan profile anyway

Reply:
I am fine either way, we don't use the filesystem anymore so we have no need to upgrade.

Reply:
I upgraded in one of the other PRs to be consistent with the conan profiles